test/suites/config.sh

```diff
@@ -207,7 +207,7 @@ test_config_profiles() {
 
   lxc delete foo
 
-  lxc init testimage foo
+  lxc init testimage foo -s "lxdtest-$(basename "${LXD_DIR}")"
```

Comment:
The `unconfined` profile the container is attached to does not have a root device set.